skimage/transform/radon_transform.py

```diff
@@ -188,22 +188,22 @@ def _get_fourier_filter(size, filter_name):
     return fourier_filter[:, np.newaxis]
 
 
-def iradon(radon_image, theta=None, output_size=None,
-           filter="ramp", interpolation="linear", circle=True):
+def iradon(radon_image, theta=None, output_size=None, filter="ramp",
+           interpolation="linear", circle=True, preserve_range=False):
```

Comment:
A test for the new parameter would go a long way. Otherwise, maybe just remove it as part of this PR

Reply:
Thinking about a test for this parameter, I am skeptical about if we should modify users inputs values in this function...
I would rather just care about output dtype with respect with input dtype. What do you think?

Reply:
I modified this PR in order to not modify users input data range. I removed the use of `convert_to_float` and introduced instead the `dtype` parameter. See the new description of the PR for details.